variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "org_name" {
+  type        = string
+  description = "The name of the org as registered in Bridgecrew console"
+}
+
+variable "bridgecrew_token" {
+  type        = string
+  description = "Your authentication token as can be found in https://www.bridgecrew.cloud/integrations/azure-api-access"
+}
+
+variable "base_url" {
+  type        = string
+  description = "Bridgecrew API base url"
+  default     = "https://www.bridgecrew.cloud"
```

Comment:
@livnoni @guy-suli This will be exposed in the registry. I don't want to expose it. When debugging, you need to go in and change it - life is tough

Reply:
why?? what is the different between var with default between write it hard coded in main.tf?
We did it in checkov too.
it not only debug, If I want to deploy a stage env with azure (for integration test) I can't do it with hard coded url

Reply:
Unlike pip, the registry exposes our inputs to the world. That's why I don't want to expose it and let people modify it / play with it. From tf - it doesn't matter, but for the registry - it matters. We're a long way from running tests on this

Reply:
This is an example of what I mean:
https://registry.terraform.io/modules/bridgecrewio/bridgecrew-gcp-read-only/google/0.3.1?tab=inputs
I didn't create this page - terraform generates it based on the variables declared

Reply:
@nimrodkor @guy-suli - I removed the base_url variable - set it hard coded